wal/src/component/root.rs

```diff
@@ -0,0 +1,115 @@
+use crate::virtual_dom::VNode;
+
+use super::{behavior::Behavior, Component};
+
+pub trait RootComponent: Sized {
```

Comment:
jakis skeczi shit to jest, że RootComponent i Component to są tak podobne rzeczy. Fajnie by było coś z tym zrobić nie mam pomysłu co na razie

Reply:
Ta, najlepiej jakby się dało używać zwykłego komponentu w add_page, to by było clean af